Seagate/cortx-utils
# Fixed py-utils unittests in main branch

Signed-off-by: Veerendra Garikipati <veerendra.garikipati@seagate.com>

# Problem Statement
- [Py-utils] - Fix py-utils test failures in main branch

# Design
-  [CORTX-30743](https://jts.seagate.com/browse/CORTX-30743)

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing
- Are test cases updated along with code changes due to Enhancements/Bugs [Y/N]: Y
- Confirm that new test cases are added to regression and sanity plan files and relevant feature plan files [Y/N]: Y
- Confirm that Test Cases are added for new features added [Y/N]: 
- Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- Confirm Testing was performed with installed RPM/K8s deployment [Y/N]: Y

# Review Checklist 
####  Before posting the PR please ensure:

# KVstore/Confstore feature (changes/fixes) checklist
#### Confirm changes are made for:

# Documentation
